tensorrt_llm/layers/embedding.py

```diff
@@ -177,6 +177,9 @@ def forward(self, tokens, prompt_embedding_table, tasks, task_vocab_size):
 
         # tasks: [batch_size, seq_len]
         # prompt_tokens: [batch_size, seq_len]
+        # if speculative decoding is enabled tasks is [batch_size, seq_len + max_draft_len],
```

Comment:
A little confused the explanation here. Should it be
>         # if speculative decoding is enabled, the shape of prompt_tokens is [batch_size, seq_len + max_draft_len]

Otherwise, the shape of task is already `[batch_size, seq_len + max_draft_len]`, why do we need to expand it?

Reply:
Yes, you are right

Reply:
Pushed a fix